input/fsh/Terminology.fsh

```diff
@@ -155,13 +155,35 @@ Description: "Patient's documented history of comorbidities"
 * include SACQPatientComorbidityCodeSystem#13 "Rheumatoid arthritis"
 * include SACQPatientComorbidityCodeSystem#14 "Other medical problems"
 
-ValueSet: LateralityVS
-Id: LateralityVS
-Title: "Laterality of breast cancer"
-Description: "Valueset of the laterality of breast cancer"
-* SCT#80248007 "Left breast structure"
-* SCT#73056007 "Right breast structure"
-* SCT#63762007 "Both breasts"
+ValueSet: BodyWeightVS
+Id: BodyWeightVS
+Title: "Units of patient's body weight"
+Description: "Valueset of the unit  of the patient's body weight"
+* UCUM#kg "kg"
+* UCUM#[lb_av] "[lb_av]"
+
+// ValueSet: BodyHeightVS
+// Id: BodyHeightVS
+// Title: "Units of patient's body height"
+// Description: "Valueset of the unit  of the patient's body height"
+// * UCUM#cm "cm" 
+// * UCUM#[in_i] "in_i"
+// We probalby need this VS for the questionnaires, so I'll leave it here for now
+
+// ValueSet: LateralityVS
```

Comment:
Lets take this out (and anything else we don't need), we can always look back in Git's history should we need it. If we leave it in, we won't be sure why did we decide to leave it in when we're cleaning things up.